src/java.base/share/classes/sun/security/ssl/ProtocolVersion.java

```diff
@@ -290,7 +290,7 @@ static boolean isNegotiable(
             ProtocolVersion pv = ProtocolVersion.nameOf(pn);
             if (pv == null) {
                 throw new IllegalArgumentException(
-                        "Unsupported protocol" + pn);
+                        "Unsupported protocol " + pn);
```

Comment:
I think you should also add a colon so this reads better, i.e.:  `"Unsupported protocol: " + pn);`